examples/README.md

```diff
@@ -629,6 +630,28 @@ The blueprint file is composed of 3 primary parts, top-level parameters,
 deployment variables and deployment groups. These are described in more detail
 below.
 
+### Blueprint Boilerplate
+
+The following is a template that can be used to start writing a blueprint from
+scratch.
+
+```yaml
```

Comment:
I could see a good argument for creating a separate file for this in examples, that makes it one step easier to start creating a blueprint from scratch by either copying the file or just working in the template directly.

That could also be added at a later time, so I won't hold up the PR on it.

Reply:
Ack.